nova/compute.sls

```diff
@@ -143,8 +143,21 @@ Add_compute_to_availability_zone_{{ compute.availability_zone }}:
   - name: "nova {{ identity_params }} aggregate-add-host {{ compute.availability_zone }} {{ pillar.linux.system.name }}"
   - unless: "nova {{ identity_params }} service-list | grep {{ compute.availability_zone }} | grep {{ pillar.linux.system.name }}"
 
+{%- if compute.aggregates != None %}
+
+{%- for aggregate in compute.aggregates %}
+Add_compute_to_aggregate_{{ aggregate }}:
```

Comment:
my experience shows that it isn't the best place to do it: when the state is applied for the first time, it will almost surely fail because the compute service isn't registered in the Nova API yet. I think it would be more relevant to let it manage by the nova.client role.